tests/acceptance/features/apiSharingNg/listPermissions.feature

```diff
@@ -196,3 +196,27 @@ Feature: List a sharing permissions
         }
       }
       """
+
+
+  Scenario: user send share invitation for all allowed roles defined in permission lists for a file
+    Given user "Alice" has uploaded file with content "hello text" to "textfile.txt"
+    And user "Brian" has been created with default attributes and without skeleton files
+    When user "Alice" gets permissions list for file "textfile.txt" of the space "Personal" using the Graph API
+    Then the HTTP status code should be "200"
+    And user "Alice" should be able to send share invitation with all allowed permission roles from the above response:
```

Comment:
```suggestion
    And user "Alice" should be able to send share invitation with all allowed permission roles
```
I think it might not be necessary to mention `from the above response`